ironfish-contracts/contracts/Ironfish.sol

```diff
@@ -2,11 +2,12 @@
 pragma solidity ^0.8.18;
 
 contract Ironfish {
```

Comment:
Can we move this to the sdk? If we do this we will be able to export and use directly in the rest of the sdk and cli

Reply:
Yeah I can move this to a separate PR